src/components/Navbar/Navbar.js

```diff
@@ -35,10 +57,24 @@ const Navbar = () => {
             <span>Store</span>
           </Link>
         </div>
-        <div className="search_box">
-          <input type="search" placeholder="Search Here..." />
-          <span className="fa-solid fa-magnifying-glass"></span>
-        </div>
+
+        {location.pathname === "/products" && (
+          <form
+            onSubmit={(e) => searchForProduct(e, searchInput)}
+            className="search_box"
+          >
+            <input
+              type="search"
+              placeholder="Search Here..."
+              value={searchInput}
+              onChange={(e) => setSearchInput(e.target.value)}
+            />
+            <button
+              type="submit"
+              className="fa-solid fa-magnifying-glass btn search-btn"
+            ></button>
+          </form>
+        )}
 
         {encodedToken ? (
```

Comment:
suggestion: you can use a global boolean state to check if the user is loggedIn or not, instead of directly using token